packages/starlight/__tests__/basics/navigation-labels.test.ts

```diff
@@ -59,6 +51,14 @@ describe('getSidebar', () => {
 			    "label": "guides",
 			    "type": "group",
 			  },
+			  {
+			    "attrs": {},
+			    "badge": undefined,
+			    "href": "/",
+			    "isCurrent": true,
+			    "label": "Home Page",
+			    "type": "link",
+			  },
```

Comment:
Noting that this change moves `index` files down to the bottom here. We haven’t really decided/documented what sorting is like for `index`, but I’d kind of assume they should come first. 🤔

Reply:
Indeed, `index` stands out as a particular case when sorting by `filename`. 

Considering this, using the frontmatter title could offer more control over the sorting sequence and aesthetically aligns with the sidebar's order, given that the sidebar label relies on the frontmatter title. 

However, it's essential to note the potential tradeoffs, such as the scenario where the title significantly differs 😆 from the slug mentioned by @HiDeoo. Another concern is that each i18n variation could present a distinct order based on its language spelling, which might change the intended order set by the default lang.